fitbenchmarking/utils/options.py

```diff
@@ -340,9 +360,12 @@ def list_to_string(mylist):
                                  self.algorithm_type),
                              'software': list_to_string(self.software),
                              'jac_method': list_to_string(self.jac_method),
+                             'hes_method': list_to_string(self.hes_method),
                              'max_runtime': self.max_runtime}
         config['JACOBIAN'] = {k: list_to_string(m)
                               for k, m in self.num_method.items()}
```

Comment:
I'm wondering if this should be renamed to `jac_num_method`?

Reply:
That would be a better name.